<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>myMainWindow</class>
 <widget class="QMainWindow" name="myMainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1437</width>
    <height>727</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>myMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">* {
	background:qlineargradient(spread:pad, x1:0.465636, y1:1, x2:0.465, y2:0, 								stop:0  rgba(48, 61, 66, 255), stop:1 rgba(54, 77, 96, 255)); 
	color: rgb(255, 255, 255);
}

QPlainTextEdit {
		background:qlineargradient(spread:pad, x1:0.465636, y1:1, x2:0.465, y2:0, stop:0 rgba(54, 77, 96, 255), stop:1 rgba(48, 61, 66, 255)); 
		color: rgb(153, 247, 255)
}

QPlainTextEdit:focus{ 
		 border: 2.5px inset coral;
 }

QTreeWidget::item{
/**/
background-color:transparent;
color:rgb(255, 255, 255);
}

QTreeWidget::item:hover{
/**/

}

QTreeWidget::item:selected {
/**/

}

QTreeView::branch{
    background-color:transparent;

}
QTreeView::branch:open{
	image: url(:/img/nlyl-orange-triangle_90.png);
}
QTreeView::branch:closed:has-children{
	image: url(:/img/nlyl-orange-triangle.png);
}

QTreeWidget:focus{  
	border: 2.5px inset coral
}

QToolTip{
	color:rgb(255, 255, 255);
}

QTreeWidget{
	background:qlineargradient(spread:pad, x1:0.465636, y1:1, x2:0.465, y2:0, stop:0  rgba(54, 77, 96, 255) , stop:1 rgba(48, 61, 66, 255)); 
	color:rgb(255, 255, 255);
}

QToolBar{
	padding: 4px; 
}

QToolButton{ 
	border: 3px outset rgb(76, 107, 130); 
}
QToolButton:hover{  
	background-color:rgb(255, 85, 0);  
	border: 2.5px outset coral; 
}
QToolButton:pressed{  
	border: 2.5px inset coral;
 }
QToolButton:!enabled{
	border: 2.5px inset grey;
}

QHeaderView::section {
	min-height: 20px;
 	background-color: qlineargradient(x1:0, y1:1, x2:0, y2:0,
    														stop:0 #505050, stop: 1 #616161);
   	border: 1px solid #6c6c6c;
	color: white;
	font: 10pt;
}



Line { 
	background-color: 1; 
}

QTextCharFormat{ border-radius: 10px; }</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0,0,0">
    <item row="3" column="1" colspan="2">
     <widget class="myPlainTextEdit" name="plainTextEdit">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <stylestrategy>PreferAntialias</stylestrategy>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="myCheckBox" name="checkBox">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QTextEdit" name="textEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;!- Вставьте сюда путь к сценарию --&amp;gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&amp;lt;!- Вставьте сюда описание сценарию --&amp;gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>false</underline>
        <kerning>true</kerning>
       </font>
      </property>
      <column>
       <property name="text">
        <string notr="true">Сценарий</string>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
        </font>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">*{
background-color: rgb(0,0,0,0);
}</string>
      </property>
      <property name="text">
       <string>Включить служебную информацию:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="5">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QGraphicsView" name="viewAlgth">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1437</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFile"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionDel"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionTask"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="separator"/>
   <addaction name="actionFile_2"/>
   <addaction name="separator"/>
   <addaction name="actionOpenScen"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionTask">
   <property name="text">
    <string>&lt;Task&gt;</string>
   </property>
   <property name="shortcutVisibleInContextMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&lt;Delete&gt;</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&lt;Copy&gt;</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>&lt;Start&gt;</string>
   </property>
  </action>
  <action name="actionFile">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Открыть сценарий</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="text">
    <string>Удалить действие</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="actionFile_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&lt;File&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;File&gt;</string>
   </property>
  </action>
  <action name="actionOpenScen">
   <property name="text">
    <string>&lt;OpenScen&gt;</string>
   </property>
   <property name="toolTip">
    <string>&lt;OpenScen&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>myplaintextedit.h</header>
  </customwidget>
  <customwidget>
   <class>myCheckBox</class>
   <extends>QCheckBox</extends>
   <header>mycheckbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
